build_info.json

```diff
@@ -298,14 +298,14 @@
   "binaries": {
     "kubectl-mongodb": {
       "patch": {
-        "s3-store": "s3://kubectl-mongodb/dev",
+        "s3-store": "mongodb-kubernetes-dev",
         "platforms": [
           "linux/amd64"
         ]
       },
       "staging": {
-        "sign": true,
-        "s3-store": "s3://kubectl-mongodb/staging",
+        "sign": false,
```

Comment:
how we're going to promote the binary if it's not signed?

Reply:
answer: we don't notarize in dev/staging due to flakiness/rate limiting of the external notarization service.